src/components/flat-table/flat-table.component.tsx

```diff
@@ -36,8 +36,8 @@ export interface FlatTableProps extends MarginProps {
   size?: "compact" | "small" | "medium" | "large" | "extraLarge";
   /** Applies max-height of 100% to FlatTable if true */
   hasMaxHeight?: boolean;
-  /** If `false` the border color of the first and last `FlatTableCell`'s in every `FlatTableRow` will be made transparent */
-  horizontalBorders?: boolean;
+  /** Controls whether to render table cells with visible vertical borders. */
```

Comment:
suggestion: it might be worth adding something about how when false it will hide the outer borders of the table (border left for first column and border right for last column etc)